app/controllers/trn_requests_controller.rb

```diff
@@ -9,18 +9,37 @@ def update
     redirect_to root_url unless trn_request
     session[:form_complete] = false
 
-    trn_request.update(trn_request_params)
-    ZendeskService.create_ticket!(trn_request)
-    redirect_to helpdesk_request_submitted_url
+    update_trn_request
+    unless FeatureFlag.active?(:use_dqt_api)
+      create_zendesk_ticket_and_redirect
+      return
+    end
+
+    begin
+      response = DqtApi.find_trn!(trn_request)
+      trn_request.update(trn: response['trn'])
+      redirect_to trn_found_path
+    rescue DqtApi::ApiError, Faraday::TimeoutError, DqtApi::TooManyResults
+      create_zendesk_ticket_and_redirect
```

Comment:
Do you this helper method is worth it when we're only duplicating it in two spots?

I think I would prefer to explicitly see the `redirect_to` calls inside this controller method. :thinking: 

When controller methods get long, I think the best pattern is to extract services/libs from what you can, and leave `redirect_to` and similar control flow calls inside the controller methods for clarity.

This is a non-blocking comment.

Reply:
I did this because of a Rubocop warning. I can remove the helper method and modify the Rubocop rule instead.